command/polybftmanifest/manifest_init.go

```diff
@@ -125,28 +134,47 @@ type manifestInitParams struct {
 	manifestPath         string
 	validatorsPath       string
 	validatorsPrefixPath string
-	premineValidators    string
+	premineBalanceRaw    string
+	premineBalance       *big.Int
+	stakeRaw             string
+	stake                *big.Int
 	validators           []string
 	chainID              int64
 }
 
 func (p *manifestInitParams) validateFlags() error {
-	if _, err := os.Stat(p.validatorsPath); errors.Is(err, os.ErrNotExist) {
+	var (
+		stake          *big.Int
+		premineBalance *big.Int
+		err            error
+	)
+
+	if _, err = os.Stat(p.validatorsPath); errors.Is(err, os.ErrNotExist) {
 		return fmt.Errorf("provided validators path '%s' doesn't exist", p.validatorsPath)
 	}
 
-	if _, err := types.ParseUint256orHex(&p.premineValidators); err != nil {
-		return fmt.Errorf("invalid premine validators balance provided '%s': %w", p.premineValidators, err)
+	if premineBalance, err = types.ParseUint256orHex(&p.premineBalanceRaw); err != nil {
+		return fmt.Errorf("invalid premine validators balance provided '%s': %w", p.premineBalanceRaw, err)
 	}
 
+	if p.stakeRaw != "" {
+		if stake, err = types.ParseUint256orHex(&p.stakeRaw); err != nil {
+			return fmt.Errorf("invalid stake amount provided '%s': %w", p.premineBalanceRaw, err)
```

Comment:
`stakeRaw` instead of `premineBalanceRaw`

Reply:
Fixed https://github.com/0xPolygon/polygon-edge/pull/1137/commits/606009d9a1f8e792852493a77de44b7fa9c6ae6c